android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BeaconConfigFragment.java

```diff
@@ -330,6 +330,12 @@ private void saveEditCardUrlToBeacon() {
     // Shorten the url if necessary
     if (ConfigUriBeacon.uriLength(url) > ConfigUriBeacon.MAX_URI_LENGTH) {
       url = UrlShortener.shortenUrl(url);
+      // If url shortening failed
+      if (url == null) {
+        Toast.makeText(getActivity(), R.string.url_shortening_error, Toast.LENGTH_SHORT).show();
+        // Exit here
```

Comment:
I don't think this comment is necessary. The return statement makes it pretty obvious.